marko.json

```diff
@@ -178,6 +178,31 @@
     "@pressed": "boolean",
     "@aria-selected-text": "string"
   },
+  "<ebay-filter-menu>": {
+    "renderer": "./src/components/ebay-filter-menu/index.js",
+    "@*": "expression",
+    "@html-attrbibutes": "expression",
+    "@variant": "string",
+    "@text": "string",
```

Comment:
`text` isn't a valid option on ebay-filter-menu, right?  Its just for the button.  The examples also have text being passed.

Reply:
Great catch. Yes, the text isn't needed at all.